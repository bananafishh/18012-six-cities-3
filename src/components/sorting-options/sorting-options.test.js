import React from 'react';
import renderer from 'react-test-renderer';

import SortingOptions from './sorting-options';

const sortingOptions = [
  {
    label: `Popular`,
    value: `popular`,
  },
  {
    label: `Price: low to high`,
    value: `priceLowToHigh`,
  },
];

it(`Компонент «SortingOptions» рендерится корректно`, () => {
  const tree = renderer
    .create(
        <SortingOptions
          options={sortingOptions}
          currentOption={sortingOptions[1]}
          onOptionChange={() => {}}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
