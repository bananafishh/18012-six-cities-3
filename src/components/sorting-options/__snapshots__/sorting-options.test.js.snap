// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент «SortingOptions» рендерится корректно 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by
  </span>
   
  <span
    className="places__sorting-type"
    onClick={[Function]}
    tabIndex="0"
  >
    Price: low to high
    <svg
      className="places__sorting-arrow"
      height="4"
      width="7"
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
  <ul
    className="places__options places__options--custom"
  >
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex="0"
    >
      Popular
    </li>
    <li
      className="places__option places__option--active"
      onClick={[Function]}
      tabIndex="0"
    >
      Price: low to high
    </li>
  </ul>
</form>
`;
